test/integrations/destinations/iterable/router/data.ts

```diff
@@ -541,28 +550,24 @@ export const data = [
                 version: '1',
                 type: 'REST',
                 method: 'POST',
-                endpoint: 'https://api.iterable.com/api/users/bulkUpdate',
+                endpoint: 'https://api.iterable.com/api/users/update',
                 headers: {
                   'Content-Type': 'application/json',
                   api_key: '62d12498c37c4fd8a1a546c2d35c2f60',
                 },
                 params: {},
                 body: {
                   JSON: {
-                    users: [
```

Comment:
Can you check why did the JSON is not a nested `users` object anywhere, is there any change in the implementation?

Reply:
done